vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-testbench/src/main/java/com/vaadin/flow/component/spreadsheet/testbench/SheetCellElement.java

```diff
@@ -69,7 +72,7 @@ public boolean isCellSelected() {
      *         cell has e.g. a custom editor component.
      */
     public boolean isNormalCell() {
-        List<WebElement> children = findElements(By.xpath(".//*"));
+        List<WebElement> children = findElements(By.cssSelector("*"));
```

Comment:
Xpath selector doesn't work inside shadow DOM so changed all instances to use CSS selector instead